src/js/disability-benefits/utils/helpers.js

```diff
@@ -59,37 +59,51 @@ export function getUserPhase(phase) {
   return phase - 3;
 }
 
+export function getItemDate(item) {
+  if (item.receivedDate) {
+    return item.receivedDate;
+  } else if (item.documents && item.documents.length) {
+    return item.documents[item.documents.length - 1].uploadDate;
+  } else if (item.type === 'other_documents_list' && item.uploadDate) {
+    return item.uploadDate;
+  }
```

Comment:
On the backend, the date attribute is set by going through closed_date received_date opened_date requested_date and suspense_date (in that order) and returning the first one that is non-nil. I dunno if your work is a bit redundant here. Or if it is different, we should change the backend to match the behavior of the front-end because we sort by the `date` field.